### PR TITLE
Meta: Enable the brand new triagebot transfer command

### PR DESCRIPTION
**Pull request by fmease**
cc rust-lang/triagebot#1780, rust-lang/rust-forge#741
cc @ehuss, @jackh726

### CODE REVIEW

File: triagebot.toml [hunk 1]
```diff
@@ -1057,3 +1057,7 @@ project-exploit-mitigations = [
 # Enable tracking of PR review assignment
 # Documentation at: https://forge.rust-lang.org/triagebot/pr-assignment-tracking.html
 [pr-tracking]
+
+# Enable issue transfers within the org
+# Documentation at: https://forge.rust-lang.org/triagebot/transfer.html
```
Comment (fmease) [inline triagebot.toml:1062]: Location doesn't exist yet, depends on rust-lang/rust-forge#741. The URL is unlikely to change I'd say, so not blocking imo.

**Comment by matthiaskrgr** [comment]
:eyes:  I'm wondering what happens if we try to transfer a pullrequest.

**Comment by fmease** [comment]
It errors

**Comment by Kobzol** [comment]
This PR has been merged by accident, without the full test suite being run. It was my fault (https://github.com/rust-lang/rust/pull/124631 got merged and it broke our CI). @fmease Could you please create a new PR with your branch against the latest version of `origin/master`? Thank you, and sorry for the mess.

**Comment by dtolnay** [comment]
New PR: #124768

**Pull request merged**